src/config/graphical-session/kde.md

```diff
@@ -0,0 +1,17 @@
+# KDE
+
+## Installation
+
+Install the `kde5` package, and optionally, the `kde5-baseapps` package.
+
+To use the "Networks" widget, enable the `dbus` and `NetworkManager` services.
+
+Installing the `kde5` package also installs the `sddm` package, which provides
+the `sddm` service for the Simple Desktop Display Manager; [test the
+service](../services/managing.html#testing-services) before enabling it. If you
+are not intending to run SDDM via a remote X server, you will need to install
+either the `xorg-minimal` package or the `xorg` package. SDDM defaults to
+providing an Xorg session, but a Wayland session is available.
```

Comment:
it's not clear if SDDM is itself using Wayland/Xorg (like GDM does) or if it can load a Plasma Wayland or Xorg session. I'd use "Plasma" instead of "SDDM", because you can load those sessions from any DM (or no DM at all).

Reply:
@ericonr: Ah, yes, point taken. SDDM itself needs an X server (cf. https://github.com/void-linux/void-packages/issues/21862). Basically what i'm trying to get at is: "SDDM will start an Xorg-based Plasma session by default, but you can also start a Wayland-based Plasma session." Does that make more sense? If so, i'll try to find better wording along those lines.

Reply:
I think that works.

Reply:
Thanks, PR updated.